pkg/pool-weighted/contracts/lbp/LBPool.sol

```diff
@@ -302,6 +355,16 @@ contract LBPool is WeightedPool, Ownable2Step, BaseHooks {
         uint256[] memory startWeights,
         uint256[] memory endWeights
     ) internal virtual {
+        InputHelpers.ensureInputLengthMatch(_NUM_TOKENS, startWeights.length);
+        InputHelpers.ensureInputLengthMatch(_NUM_TOKENS, endWeights.length);
+
+        if (endWeights[0] < _MIN_WEIGHT || endWeights[1] < _MIN_WEIGHT) {
+            revert MinWeight();
+        }
+        if (endWeights[0] + endWeights[1] != FixedPoint.ONE) {
+            revert NormalizedWeightInvariant();
+        }
```

Comment:
I think this is all checked inside the `WeightedPool` constructor. This was only needed for the external method, given that parameters needed validation. But if we're only calling this inside the constructor it shouldn't be needed.

Reply:
Let's leave it anyways. Doesn't hurt really, it's only called once.